lib/game_runner.rb

```diff
@@ -1,34 +1,34 @@
-class GameRunner
-  attr_reader :stdout
+require_relative "grid"
+require_relative "printer"
 
-  def grid_template
-    <<~GRID
-       1  2  3
-      __ __ __
-   A |  |  |  |
-     |__|__|__|
-   B |  |  |  |
-     |__|__|__|
-   C |  |  |  |
-     |__|__|__|
-    GRID
-  end
+class GameRunner
+  attr_reader :stdout, :stdin, :grid, :printer
 
-  def initialize(stdout, stdin)
+  def initialize(stdout, stdin, grid = Grid.new, printer = Printer.new(stdout, grid))
```

Comment:
I like initializing with the grid, but the printer should go in a method. 

The reason I like the grid with a default is because it will allow at some point dependency injecting it in tests to get the app in a certain moment.